apps/balancer-tools/src/app/poolsimulator/(components)/ShareButton.tsx

```diff
@@ -21,7 +21,7 @@ export function ShareButton() {
             : "Copy a link with the current simulation parameters"
         }
       >
-        <Button shade="dark" disabled={onInitialPage}>
+        <Button shade="dark" disabled={true}>
```

Comment:
Do we need to come back to this later? If so I'd suggest we add a TODO with a task reference here.

Reply:
Ow, I think that was a problem dealing with conflicts. Luiza had change this file and change It back later.